components/brave_rewards/common/mojom/rewards.mojom

```diff
@@ -412,16 +412,17 @@ struct ExternalWallet {
   string token;
   string address;
   WalletStatus status;
-  string one_time_string;
-  string code_verifier;
   string user_name;
   string member_id;
   string account_url;
-  string login_url;
   string activity_url;
   map<string, double> fees;
 };
 
+struct ExternalWalletLoginParams {
+  string url;
+};
```

Comment:
Do we need a separate Mojom struct for this URL?

Reply:
The intension here is to leave room for `BeginLogin` to return headers (or cookies) to support other login flows.